amd64/src/semantic.rs

```diff
@@ -747,7 +755,7 @@ pub fn call(a: Rvalue) -> Result<(Vec<Statement>, JumpSpec)> {
         call ?, new_rip:64;
     });*/
     let stmts = rreil!{
-        call ?, (a);
```

Comment:
Of course this is somewhat pre-mature optimization, but I really am convinced the binary format will make things previously impossible, possible, e.g., analyzing a 20-30MB binary, etc.